src/stylus/elements/_global.styl

```diff
@@ -27,10 +27,20 @@ main
   will-change: padding-left
 
 .application--light a
-  color: $theme.primary
+  color: $material-light.text.link
```

Comment:
should the styles in this file for application go here? https://github.com/vuetifyjs/vuetify/blob/dev/src/stylus/components/_app.styl